bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| helios-ventilation-easycontrols | Helios Heat-Recovery Ventilation devices with easyControls |
+
+## Configuration
+You first need to set up a Modbus bridge according to the Modbus documentation. Things in this extension will use the selected bridge to connect to the device. The configuration of a Helios Ventilation device via a `.things` file would look like the following code sample. It's required to provide the device's IP address, port and unit ID (port and unit ID cannot be changed and always have to be 502 and 180; in fact the values provided here are ignored by the binding and the fixed values 502 and 180 are used):
+```
+Bridge modbus:tcp:modbus-gateway "Modbus TCP/IP Gateway" [ host="x.x.x.x", port=502, id=180, enableDiscovery=true ] {
+    Thing helios-ventilation-easycontrols kwl "KWL"
+}
+```
+
+## Channels
+The following channels are supported:
+
+| Channel                          | Channel Group   | Description                                                      | Item Type                | Unit | Values                                                                                               |
+| -------------------------------- | --------------- | ---------------------------------------------------------------- | ------------------------ | ---- | ---------------------------------------------------------------------------------------------------- |
+| articleDescription               | general         | The KWL's article description                                    | String                   |      |                                                                                                      |
+| refNo                            | general         | The KWL's reference number                                       | String                   |      |                                                                                                      |
+| macAddress                       | general         | The KWL's MAC Address                                            | String                   |      |                                                                                                      |
+| language                         | general         | The KWL user interface's language                                | String                   |      | de, en, fr, hr, hu, it, pl, sk, sl                                                                   |
+| date                             | general         | The KWL's system date                                            | DateTime                 |      |                                                                                                      |
+| time                             | general         | The KWL's system time                                            | DateTime                 |      |                                                                                                      |

```

Comment:
Is there any reason why these Channels are separated?

Reply:
The reason is that these are separate variables in the device's Modbus protocol, i.e. they can be changed separate from one another.

Reply:
It would be good usability if the user could use one `DateTime` command to change it and the binding translates it into two values.